clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -311,15 +446,24 @@
     )
 
     (asserts! (<= percent u100) (err u5))
+    (asserts!
+      (and
+        (is-eq (unwrap-panic (contract-call? .arkadiko-dao get-emergency-shutdown-activated)) false)
+        (is-eq (var-get swap-shutdown-activated) false)
+      )
+      (err ERR-EMERGENCY-SHUTDOWN-ACTIVATED)
+    )
+    (asserts! (is-eq (get enabled pair) true) (err ERR-PAIR-DISABLED))
```

Comment:
```suggestion
    (asserts! (get enabled pair) (err ERR-PAIR-DISABLED))
    (asserts!
      (and
        (is-eq (unwrap-panic (contract-call? .arkadiko-dao get-emergency-shutdown-activated)) false)
        (is-eq (var-get swap-shutdown-activated) false)
      )
      (err ERR-EMERGENCY-SHUTDOWN-ACTIVATED)
    )
```
Verifying pair is enabled should be done before verifying emergency shutdown. That way if pair is disabled TX will consume less execution costs and as a result will leave more room for other transactions.

Also checking if `bool` value is equal `true` is a bit excessive.